### PR TITLE
Sand Tetris

### LINKED ISSUE #4692: [New game]: Sand Tetris
### 🎮 Game Request

Game logic and basic description
A block of sand comes down which can be controlled by arrow keys, upon contacting ground the block breaks and behaves like sand.
The player tries to match the sand colors and connect both the side walls and if done, he scores points.
If sand reaches top position, Game Over

### Point down the features

Game points
Connect both sidewalls using same color of sand to score point

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Markcosay**
## PR Description 📜

Adding new Game, Sand Tetris. Works like regular tetris but blocks are made of sand and sand physics works on them.
Fixes #4692 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in GameZone's README.md
- [x] I have added README.md in my Game folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.
- [x] I have added my Game to gamesData.json file and it is loading properly on main website as intended.

<hr>

## Add your screenshots(Optional) 📸

screenshot_sand_tetris

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Markcosay ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by Markcosay** [comment]
@ishita-43 Please check

**Comment by ishita-43** [comment]
Resolve arising merge conflict in gamesdata.json

**Comment by Markcosay** [comment]
@ishita-43 done

File: README.md [hunk 1]
```diff
@@ -203,7 +203,12 @@ This repository also provides one such platforms where contributers come over an
...
+
+| [Emoji_Intruder](https://github.com/kunjgit/GameZone/tree/main/Games/Emoji_Intruder)                             | [Guess The Weapon](https://github.com/kunjgit/GameZone/tree/main/Games/Guess_The_Weapon)                | [Guess Who](https://github.com/kunjgit/GameZone/tree/main/Games/Guess_Who)                                   | [Pop My Balloon](https://github.com/kunjgit/GameZone/tree/main/Games/Pop_My_Balloon)                               | [Tower Stack](https://github.com/kunjgit/GameZone/tree/main/Games/Tower_Stack) | [Sand Tetris](https://github.com/markcosay/GameZone/tree/main/Games/sand_tetris)    |                                                                                                    |
```
Comment (ishita-43) [inline README.md:207]: Change the name of repo as kunjgit instead of your repo name

**Review by ishita-43 — COMMENTED** [review]
Please make following change

File: README.md (hunk 1 above)
Comment (Markcosay) [reply README.md]: @ishita-43 Done

**Review by ishita-43 — APPROVED** [review]
Good to go

**Comment by kunjgit** [comment]
Thank you  @Markcosay , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**